data/han2020sequential/han2020sequential.yaml

```diff
@@ -0,0 +1,61 @@
+# yaml-language-server: $schema=../.schema.yaml
+title:
+  Sequential Analysis of Viral Load in a Neonate and Her Mother Infected With
+  Severe Acute Respiratory Syndrome Coronavirus 2
+description:
+  The study reports SARS-CoV-2 viral loads in different specimen types for a neonate and
+  her mother diagnosed on 2020-03-20. The study includes nasopharyngeal, oropharyngeal,
+  stool, plasma, saliva, and urine samples. Only concentrations in stool samples are
+  reported here. Viral loads were extracted manually from Figure 1 using
+  [WebPlotDigitizer](https://automeris.io).
+doi: 10.1093/cid/ciaa447
+analytes:
+  stool_E:
+    description: Viral RNA was detected using the PowerChek 2019-nCoV real-time
+      polymerase chain reaction kit (Kogene Biotech, Seoul, Korea) for amplification of
+      the E gene and the RNA-dependent RNA polymerase region of the ORF1b gene, and
+      quantified with a standard curve that was constructed using in vitro transcribed
+      RNA provided from the European Virus Archive. [copied from manuscript]
+    limit_of_detection: 5700
+    limit_of_quantification: unknown
+    gene_target: E and ORF1b
+    biomarker: SARS-CoV-2
+    reference_event: symptom onset
+    specimen: stool
+    unit: gc/mL
+participants:
+  - attributes:
+      age: 0
```

Comment:
27 day old. Should we enter 0.074 (27 days/365 days) or 0.083 (1 month/12 months) here?

Reply:
Yes, let's do that. Can't hurt, I guess. I couldn't find an age for the mother.